src/sass/components/_path.scss

```diff
@@ -0,0 +1,95 @@
+// Path Overrides
+// ==============
+
+$color-background-path-current: $brand-background-primary;
+$color-border-path-current: $brand-background-primary;
+$color-text-path-current: $color-text-default;
+$color-background-path-active: $brand-background-primary;
+$color-border-path-activ: $brand-background-primary;
+$color-text-path-active: $color-text-default;
+$color-background-path-active-hover: $color-background-alt-inverse;
+$color-background-path-complete: $color-background-path-incomplete-hover;
+$color-background-path-complete-hover: $color-gray-6;
+
+.slds-path__nav {
+  .slds-is-current {
+    background-color: $color-background-path-current;
+
+    &:before,
+    &:after {
+      background-color: $color-background-path-current;
+      background-image: linear-gradient(
+          90deg,
+          $color-border-path-current,
+          $color-border-path-current 0.125rem,
+          transparent 0.125rem
+        ),
+        linear-gradient(
+          90deg,
+          $color-border-path-current,
+          $color-border-path-current 0.125rem,
+          transparent 0.125rem
+        ),
+        linear-gradient(
+          $color-border-path-current,
+          $color-border-path-current 0.125rem,
+          transparent 0.125rem
+        );
+    }
+
+    &:first-child {
+      border-color: $color-border-path-current;
+    }
+
+    .slds-path__link {
+      color: $color-text-path-current;
+    }
+  }
+
+  .slds-is-active {
+    background: $color-background-path-active;
+
+    &:before,
+    &:after {
+      background: $color-background-path-active;
+    }
+
+    &:first-child {
+      &:before,
+      &:after {
+        background: $color-background-path-active;
+      }
+
+      &:hover {
+        background: $color-background-path-active-hover;
+
+        &:before,
+        &:after {
+          background: $color-background-path-active-hover;
+        }
+      }
+    }
+  }
+
+  .slds-is-complete {
+    background: $color-background-path-complete;
+
+    &:before,
+    &:after {
+      background: $color-background-path-complete;
+    }
+
+    .slds-path__link {
+      color: $color-text-default;
+    }
+
+    &:hover {
+      background: $color-background-path-complete-hover;
+
+      &:before,
+      &:after {
+        background: $color-background-path-complete-hover;
+      }
+    }
+  }
+}
```

Comment:
@stacyk It looks to my like we want to show `.slds-is-won` as green, but right now we're overriding it.

https://xd.adobe.com/view/75181e97-dcf5-424c-4101-ee1a2a38ee13-0329/screen/9481424f-b901-4614-a6f1-fd4ebe7c139b/12-2

![Screenshot 2020-03-11 16 24 17](https://user-images.githubusercontent.com/552316/76460652-c6469900-63b4-11ea-80f8-f4a416179225.png)